framework/src/wifi_manager/wifi_manager.c

```diff
@@ -187,7 +181,7 @@ static pthread_cond_t g_disconn_signal = PTHREAD_COND_INITIALIZER;
 		pthread_mutex_lock(&g_disconn_lock);                    \
 		pthread_cond_wait(&g_disconn_signal, &g_disconn_lock);  \
 		pthread_mutex_unlock(&g_disconn_lock);                  \
-		nvdbg("[WM] T%d wait disconnect callback\n", getpid()); \
+		ndbg("[WM] T%d wait disconnect callback\n", getpid()); \
```

Comment:
Is this an error situation so that you changed nvdbg to ndbg..?

Reply:
This is not error case but related with timing issue. So considering the importance, made it to ndbg. Thanks for the comment.